tests/suites/test_suite_cipher.blowfish.data

```diff
@@ -302,255 +302,255 @@ BLOWFISH Try encrypting 49 bytes with no padding
 
-BLOWFISH Encrypt and decrypt 32 bytes
+BLOWFISH Encrypt and decrypt 32 bytes [#4]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf:MBEDTLS_CIPHER_BLOWFISH_CFB64:"BLOWFISH-CFB64":128:33:-1
 
-BLOWFISH Encrypt and decrypt 47 bytes
+BLOWFISH Encrypt and decrypt 47 bytes [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf:MBEDTLS_CIPHER_BLOWFISH_CFB64:"BLOWFISH-CFB64":128:47:-1
 
-BLOWFISH Encrypt and decrypt 48 bytes
+BLOWFISH Encrypt and decrypt 48 bytes [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf:MBEDTLS_CIPHER_BLOWFISH_CFB64:"BLOWFISH-CFB64":128:48:-1
 
-BLOWFISH Encrypt and decrypt 49 bytes
+BLOWFISH Encrypt and decrypt 49 bytes [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf:MBEDTLS_CIPHER_BLOWFISH_CFB64:"BLOWFISH-CFB64":128:49:-1
 
-BLOWFISH Encrypt and decrypt 0 bytes in multiple parts
+BLOWFISH Encrypt and decrypt 0 bytes in multiple parts [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:0:0:-1:0:0:0:0
 
-BLOWFISH Encrypt and decrypt 1 bytes in multiple parts 1
+BLOWFISH Encrypt and decrypt 1 bytes in multiple parts 1 [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:1:0:-1:1:0:1:0
 
-BLOWFISH Encrypt and decrypt 1 bytes in multiple parts 2
+BLOWFISH Encrypt and decrypt 1 bytes in multiple parts 2 [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:0:1:-1:0:1:0:1
 
-BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 1
+BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 1 [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:16:0:-1:16:0:16:0
 
-BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 2
+BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 2 [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:0:16:-1:0:16:0:16
 
-BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 3
+BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 3 [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:1:15:-1:1:15:1:15
 
-BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 4
+BLOWFISH Encrypt and decrypt 16 bytes in multiple parts 4 [#2]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:15:1:-1:15:1:15:1
 
-BLOWFISH Encrypt and decrypt 22 bytes in multiple parts 1
+BLOWFISH Encrypt and decrypt 22 bytes in multiple parts 1 [#4]
 depends_on:MBEDTLS_BLOWFISH_C:MBEDTLS_CIPHER_MODE_CFB
 enc_dec_buf_multipart:MBEDTLS_CIPHER_BLOWFISH_CFB64:128:15:7:-1:15:7:15:7
 
-BLOWFISH Encrypt and decrypt 22 bytes in multiple parts 1
+BLOWFISH Encrypt and decrypt 22 bytes in multiple parts 1 [#5]
```

Comment:
Please remove the number at the end of this test case and others like this and have the unified `[#n]` format instead.

Reply:
Rejected as out of scope.